infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -175,7 +183,9 @@ public void handleLogAddressSpaceRequest(CorfuMsg msg, ChannelHandlerContext ctx
     @ServerHandler(type = CorfuMsgType.TRIM_MARK_REQUEST)
     public void handleTrimMarkRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
         log.trace("handleTrimMarkRequest: received a trim mark request {}", msg);
-        r.sendResponse(ctx, msg, CorfuMsgType.TRIM_MARK_RESPONSE.payloadMsg(streamLog.getTrimMark()));
+        long trimMark = streamLog.getTrimMark();
+        log.debug("handleTrimMarkRequest: trim mark response: {}", trimMark);
```

Comment:
@WenbinZhu won't this message in debug add too much logging? which does not add much, perhaps trace would be more suitable for this one?

Reply:
Initially the first log line was in debug mode until Salva's patch changed it to trace, this patch keeps the same result as before: one debug log line for this method.  Also trim mark request is not quite offen right?